### PR TITLE
Add throttling commands

### PR DESCRIPTION
**Pull request by a1exsh**
Internal ticket: 1015

### CODE REVIEW

File: bubuku/cli.py [hunk 1]
```diff
@@ -363,5 +363,35 @@ def validate_replication(factor: int):
...
+              help='Broker id to set replication throttling for. By default throttling is set for the current broker.')
+@click.argument('BYTES_PER_SECOND', type=click.INT, required=True)
+def throttle_set(broker: str, bytes_per_second: int):
+
+    with load_exhibitor_proxy(env_provider.get_address_provider(), config.zk_prefix) as zookeeper:
+
+        broker_id = int(get_opt_broker_id(broker, config, zookeeper, env_provider))
+
+        manager = RebalanceThrottleManager(bytes_per_second)
+        manager.apply_throttle([
```
Comment (adyach) [inline bubuku/cli.py:382]: do we need first to call `remove_old_throttle_configurations`? I am not sure

File: bubuku/cli.py [hunk 2]
```diff
@@ -363,5 +363,35 @@ def validate_replication(factor: int):
...
+            (topic, partition, replicas)
+            for topic, partition, replicas in zookeeper.load_partition_assignment()
+            if broker_id in replicas
+        ])
+
+
+@throttle.command('remove-all', help='Remove all replication throttling')
```
Comment (adyach) [inline bubuku/cli.py:389]: I would be sceptic to call that method, can we have also remove for single broker? so I do set and then remove for the same broker

File: bubuku/cli.py (hunk 1 above)
Comment (a1exsh) [reply bubuku/cli.py]: No, the old state is stored in the object itself, so it is only useful when running apply/remove throttle in a loop on a single `RebalanceThrottleManager` object like rebalance does.  Here we create it anew, so there is no pre-existing state.

File: bubuku/cli.py (hunk 2 above)
Comment (a1exsh) [reply bubuku/cli.py:394]: We can give it a try, but it becomes harder than I like it.

**Comment by a1exsh** [comment]
:+1:

**Comment by adyach** [comment]
👍

**Pull request merged**